### DISCUSSION TITLE
Proposal to change GitHub project name and URL

### DISCUSSION THREAD

**Discussion by bug-or-feature** [ig-python/ig-markets-api-python-library#277] [Ideas]
I propose to change the name of this GitHub project to *trading-ig*. The current name is too long, too hard to say, too hard to remember and too hard to type. *trading-ig* is already the name of the released [project on the Python package index](https://pypi.org/project/trading-ig/). 

If we proceed, the URL of the project would become:

https://github.com/ig-python/trading-ig

In my experience, GitHub handles redirection pretty efficiently. [Their docs](https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository)

One downside is that developers would have to adjust their git remotes in their local environment. Personally I don't think that is too much effort.

Any thoughts, concerns?

Tagging contributors: @femtotrader @LangleJ @xnox @RRMXkun @oliverpolden @Filco306 @zansibal @aLeX1443 @hklang10 @Esteban108 @dfaligertwood

**Comment by RRMXkun** [comment]
It's a good idea, I always figured the name was different from pypi due to specific technical reasons with Github.

**Comment by bug-or-feature** [comment]
Not much response, so assuming all happy. Done now. I'd suggest renaming your forks too, it would be confusing otherwise. And rename any remotes as necessary. See

https://git-scm.com/docs/git-remote
